bundles/org.openhab.binding.androidtv/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -114,9 +120,10 @@
 				<label>Hostname</label>
 				<description>Hostname or IP address of the device</description>
 			</parameter>
-			<parameter name="port" type="integer">
-				<label>Port</label>
+			<parameter name="googletvPort" type="integer">
```

Comment:
Is it necesarry to rename this parameter and by that breaking backwards compatibility? There is still only one port parameter for this thing.

Reply:
I suppose we don't actually "have to" but I did for code clarity purposes.  Otherwise I'm going to end up having "port", "shieldtvPort", "philipstvPort", etc all in one file and it could be confusing as to which port "port" is really tied to. 
 Reality is, with ATV being a newer binding, and because the protocol stacks come from Google (and there is no option to change it on the device), I doubt anyone has actually configured this out of default.  I considered not even including a port option but did on the off chance someone had a weird NAT setup or something.  I'm not worried about breaking backwards compatibility, I really don't think anyone has actually ever used it.

Reply:
OK, let's leave it then.